### PR TITLE
added markup to the components

### PR DESCRIPTION
**Pull request by Keerthi-Gundla**
added markup to the components

### CODE REVIEW

File: blinklist/src/components/organisms/SearchBar.tsx [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+  placeholder?: string;
+  handleChange: () => void;
 }
 
+const SearchBar: React.FC<SearchBarProps> = (props) => {
+  const { placeholder, handleChange } = props;
+
+  return (
+    <Grid>
+      <TextField placeholder={placeholder} onChange={handleChange} />
```
Comment (tarunya882) [inline blinklist/src/components/organisms/SearchBar.tsx:14]: SearchBar will not come under organism. Can you tell why this can be considered as organism

**Review by tarunya882 — COMMENTED** [review]
Please cover all organisms and pages

File: blinklist/src/components/organisms/SearchBar.tsx (hunk 1 above)
Comment (Keerthi-Gundla) [reply blinklist/src/components/organisms/SearchBar.tsx:14]: Search functionality can be written in this organism and the text that entered in this Text Field will be stored in state.

Comment (tarunya882) [reply blinklist/src/components/organisms/SearchBar.tsx:14]: Please go through atomic structure before you create components

**Review by tarunya882 — APPROVED** [review]

**Pull request merged**